release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -959,6 +959,10 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-dev-console-bug-fixes"]
 ==== Developer Console
 
+* Previously, the {product-title} used API version `v1alpha1` for Knative serving and Eventing but the flag was not set for API version `v1beta1`. With this fix, the {product-title} supports both the API versions. (link:https://issues.redhat.com/browse/OCPBUGS-5164[*OCPBUGS-5164*])
+
+* Previously, when editing any pipeline in the {product-title} console, the correct data was not rendered in the *Pipeline builder* and *YAML view* configuration options preventing you from editing the pipeline in the *Pipeline builder*. With this update, data is parsed correctly and you can edit the pipeline using the builder. (link:https://issues.redhat.com/browse/OCPBUGS-5016[*OCPBUGS-5016*])
```

Comment:
Thanks. Updated to:

> Previously, when editing any pipeline in the {product-title} console, the correct data was not rendered in the *Pipeline builder* and *YAML view* configuration options. Because of this issue, you could not edit the pipeline in the *Pipeline builder*.